setup.cfg

```diff
@@ -63,7 +63,7 @@ install_requires =
 	pandas ~=1.0
 	pydantic ~=1.6
 	python-libsbml ==5.19.0
-	rich ~=8.0
+	rich >=8.0
 	ruamel.yaml ~=0.16
```

Comment:
I think ruamel.yaml could be relaxed as well. I don't see any issue with 0.17.

Reply:
`~=0.16` means `>=0.16;<1.0` since rich is releasing major versions pretty quickly, the same doesn't work there. So `~=8.0` means `>=8.0;<9.0` but the latest is already above 10. The API seems pretty stable, though.